app/deck/1.11.x/faqs.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Frequently Asked Questions (FAQs)
+---
+
+### I use Terraform to configure Kong, why should I care about decK?
+
+If you are using Terraform and are happy with it, you should continue to use it.
+decK covers all the problems that Terraform solves and goes beyond it:
+- With Terraform, you have to track and maintain Terraform files (`*.tf`) and
+  the Terraform state (likely using a cloud storage solution). With decK, the
+  entire configuration is stored in the YAML/JSON file(s) only. There is no
+  separate state that needs to be tracked.
+- decK can export and back up your existing Kong's configuration, meaning,
+  you can take an existing Kong installation, and have a backup, as well as a
+  declarative configuration for it. With Terraform, you will have to import
+  each and every entity in Kong into Terraform's state.
+- decK can validate if a configuration file is valid or note
+  (validate sub-command).
+- decK can quickly reset your Kong's configuration when needed.
+- decK works out of the box with {{site.base_gateway}} features like
+  Workspaces and RBAC.
+
+### Can I run multiple decK processes at the same time?
+
+NO! Please do not do this. The two processes will step on each other and
+might corrupt Kong's configuration. You should ensure that there is only
+one instance of decK running at any point in time.
+
+### Kong already has built-in declarative configuration, do I still need decK?
+
+Kong has an official declarative configuration format.
+
+Kong can generate such a file with the `kong config db_export` command, which
+dumps almost the entire database of Kong into a file.
+
+You can use a file in this format to configure Kong when it is running in
+a DB-less or in-memory mode. If you're using Kong in the DB-less mode, you
+don't really need decK.
+
+But, if you are using Kong along-with a database like Postgres or Cassandra,
```

Comment:
```suggestion
But, if you are using Kong alongside a database like Postgres or Cassandra,
```
Also, do we want to mention Cassandra here since we're deprecating its usage with Kong?

Reply:
Good catch, we shouldn't.